### DISCUSSION TITLE
Pagebuilder content video type banner support

### DISCUSSION THREAD

**Discussion by sandipansaha** [breezefront/community#153] [2. Ideas]
Hi Support,

Magento default pagebuilder banner element supports video banners. However, that does not work with Breeze. Can we please have that support.

![image](https://github.com/breezefront/community/assets/4614581/76dcec7d-141d-412d-bb48-dbea44213fde)

Thanks

**Comment by vovayatsyuk** [comment]
What version of breeze module do you use? I believe I added support for video background in 2.6.0

https://breezefront.com/updates#may-22-2023

**Reply by sandipansaha** [reply]
My apologies! I did not notice that the particular setup was outdated.
Was working on pagebuilder extend to convert the banner element render from the background image to the picture tag with overlay, and was checking the video banner at that time.

After the upgrade, it works absolutely fine.

![image](https://github.com/breezefront/community/assets/4614581/18587586-8399-4924-84c1-2a98d9b2beee)
